CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v0.9.0]
+
+### Added
+
+- SSO support
```

Comment:
d: what does SSO support mean? You mean configuring Entra ID as an SSO Provider for meshStack. Does not include Identity Lookup?

Reply:
It's this part

https://github.com/meshcloud/terraform-azure-meshplatform?tab=readme-ov-file#single-sign-on-sso-integration